NEWS.md

```diff
@@ -383,6 +383,9 @@ Deprecated or removed
     which now require a module argument.
     And it caused the bugfix of other default arguments to use the Main module (including `whos`, `which`).
 
+  * `expand(ex)` and `expand(module, ex)` have been deprecated in favor of
```

Comment:
Perhaps add a reference to #22064 and mention that `expand(ex)` is deprecated as well? There does not seem to be any news about that deprecation, and makes sense to mention them both in the same bullet point here.

Reply:
The immediately preceding news item covers #22064 :). The reference to #22064 occurs midstream rather than at the end of that news item. I will move that reference to the end of the that news item, and also add it to the news item for this pull request just for good measure :).